Notice: tag_provenance_here called on object: HH\Set in %s/collection.php on line 16
dict(2) {
  ["outer"]=>
  string(0) ""
  ["inner"]=>
  string(%d) "%s/collection.php:26"
}
dict(2) {
  ["outer"]=>
  string(0) ""
  ["inner"]=>
  string(%d) "%s/collection.php:27"
}

Fatal error: Uncaught Error: done in %s/collection.php:28
Stack trace:
%A
